frontend/src/lib/components/TaxonomicFilter/types.ts

```diff
@@ -19,6 +19,8 @@ export interface TaxonomicFilterProps {
     width?: number
     popoverEnabled?: boolean
     selectFirstItem?: boolean
+    /** use to filter results in a group by name, currently only working for EventProperties */
+    excludedProperties?: { [key in TaxonomicFilterGroupType]?: string[] }
```

Comment:
Would `Record<TaxonomicFilterGroupType, string[]>` also work? I think it'd be easier to understand.

Reply:
No, that doesn't work because it doesn't map automatically the enum keys to the record keys, unfortunately!

Reply:
Ah, got it... 😄 🤷